can-bus/json/PDM/PDM_tx.json

```diff
@@ -12,17 +12,34 @@
             }
         }
     },
-    "PDM_Errors": {
-        "msg_id": 403, 
-        "cycle_time": 1000,
-        "description": "Errors on the PDM.",
+    "PDM_CanFifoOverflow": {
+        "msg_id": 403,
+        "cycle_time": 5000,
+        "description": "PDM tx/rx fifo overflow count",
         "signals": {
             "TxOverflowCount": {
-                "bits": 8
+                "bits": 32
```

Comment:
are 8 bits sufficient? I could reduce it to 8 on the BMS as well if so

Reply:
yes 8 bits more than enough, fixing on BMS too would be great